File: Data Structures & Algorithms/duplicate-integer/submission-9.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        Map<Integer, Integer> hm = new HashMap<>();
        for(int num:nums){
            if(hm.getOrDefault(num, 0)!=0){
                return true;
            }
            hm.put(num, hm.getOrDefault(num, 0)+1);
        }
        return false;
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-9.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        dic = Counter(nums)
        items = sorted(
            dic.items(), 
            key = lambda item: item[1], 
            reverse = True 
        )
        return [num for num, frequency in items[:k]]
        
        
        